cirrus/server/Dockerfile

```
@@ -51,15 +26,15 @@ COPY poetry.lock /cirrus/
 RUN poetry config virtualenvs.create false && \
     poetry install --only=main --no-interaction --no-ansi
 
-# Third stage: Deploy stage
-FROM python:3.11.2-slim-buster as deploy
+# Second stage: Deploy stage
+FROM python:3.11.2-slim-bullseye as deploy
 
 WORKDIR /cirrus
 
-# Copy specific Rust components
-COPY --from=rust-builder /application-services/target/release/libcirrus.so /application-services/libcirrus.so
-COPY --from=rust-builder /application-services/cirrus.py /application-services/cirrus_sdk.py
-COPY --from=rust-builder /application-services/fml.py /application-services/fml_sdk.py
+# Copy cirrus megazord
+COPY --from=as-builder /application-services/cirrus/lib/libcirrus.so /application-services/libcirrus.so
+COPY --from=as-builder /application-services/cirrus/cirrus.py /application-services/cirrus_sdk.py
+COPY --from=as-builder /application-services/cirrus/fml.py /application-services/fml_sdk.py
```

I think splitting the logic for preparing the megazord between here and `fetch-application-services.sh` can be avoided, so we just have

```suggestion
COPY --from=as-builder /application-services/cirrus /application-services
```

or 

```suggestion
COPY --from=as-builder /application-services/cirrus /application-services/megazord
```